dotCMS/src/main/resources/dotmarketing-config.properties

```diff
@@ -853,6 +853,11 @@ STARTER_BUILD_VERSION=${starter.deploy.version}
 show.lts.eol.message=false
 date.lts.eol=12/31/2099
 
+
+##Api Token API Properties
+DISPLAY_EXPIRING_TOKEN_ALERTS = true
```

Comment:
I think since the default is true in code we can skip this one in the config file. 
Thoughts @gortiz-dotcms ?